temporal/api/cloud/cloudservice/v1/service.proto

```diff
@@ -112,6 +112,22 @@ service CloudService {
         };
     }
 
+    // Failover a multi-region namespace
+    rpc FailoverNamespaceRegion (FailoverNamespaceRegionRequest) returns (FailoverNamespaceRegionResponse) {
+        option (google.api.http) = {
+            post: "/api/v1/namespaces/{namespace}/failover",
```

Comment:
```suggestion
            post: "/api/v1/cloud/namespaces/{namespace}/failover-region",
```

Need different prefix, and I think we should add region to the URL instead of generic "failover" (unless there is a good case for calling this RPC `FailoverNamespace` instead of `FailoverNamespaceRegion`)